frontend/src/scenes/insights/Insights.tsx

```diff
@@ -161,119 +168,121 @@ export function Insights(): JSX.Element {
-                                Retention
-                                <InsightHotkey hotkey="r" />
-                            </span>
-                        }
-                        key={ViewType.RETENTION}
-                    />
-                    <TabPane
-                        tab={
-                            <span data-attr="insight-path-tab">
-                                User Paths
-                                <InsightHotkey hotkey="p" />
-                            </span>
-                        }
-                        key={ViewType.PATHS}
-                    />
-                    <TabPane
-                        tab={
-                            <Tooltip
-                                placement="bottom"
-                                title={
-                                    <>
-                                        Stickiness shows you how many days users performed an action repeteadely within
-                                        a timeframe.
-                                        <br />
-                                        <br />
-                                        <i>
-                                            Example: If a user performed an action on Monday and again on Friday, it
-                                            would be shown as "2 days".
-                                        </i>
-                                    </>
-                                }
-                                data-attr="insight-stickiness-tab"
-                            >
-                                Stickiness
-                                <InsightHotkey hotkey="k" />
-                            </Tooltip>
-                        }
-                        key={ViewType.STICKINESS}
-                    />
-                    <TabPane
-                        tab={
-                            <Tooltip
-                                placement="bottom"
-                                title={
-                                    <>
-                                        Lifecycle will show you new, resurrected, returning and dormant users so you
-                                        understand how your user base is composed. This can help you understand where
-                                        your user growth is coming from.
-                                    </>
-                                }
-                                data-attr="insight-lifecycle-tab"
-                            >
-                                Lifecycle
-                                <InsightHotkey hotkey="l" />
-                            </Tooltip>
-                        }
-                        key={ViewType.LIFECYCLE}
-                    />
-                </Tabs>
-            </Row>
+            <Header dashboardItem={dashboardItem} />
+            {Object.keys(dashboardItem).length === 0 && (
```

Comment:
Related to my typing comment on `insightLogic` but it would be clearer to just default this to `null` and check if the value is truthy